docs/resources/new_in_doc.md

```diff
@@ -16,7 +26,7 @@ This page contains recent highlights and notable changes in [[= product_name =]]
 
 - [v4.6.0 to v4.6.3 update instructions](https://doc.ibexa.co/en/master/update_and_migration/from_4.5/update_from_4.5/#v463)
 
-### Users
+### User management
```

Comment:
To have the same each time we refers to Users. The first "User management" was in [December 2023](https://ez-systems-developer-documentation--2389.com.readthedocs.build/en/2389/resources/new_in_doc/#user-management_2). This is the second one. Maybe a third one this May.

Reply:
In my opinion I would leave it as it is: Users, since it leads to section "Users", not "User management".

Reply:
@juskora  You're right, this section came from the chapter it targets.

```suggestion
### Users
```